vortexasdk/endpoints/vessel_movements.py

```diff
@@ -0,0 +1,113 @@
+
+            filter_origins: A geography, or list of geographies to filter on. Both geography names or IDs can be entered here.
+
+            filter_owners: An owner, or list of owners to filter on. Both charterer/owner names or IDs can be entered here.
+
+            filter_products: A product, or list of products to filter on. Both product names or IDs can be entered here.
+
+            filter_vessels: A vessel, or list of vessels to filter on. Both vessel names or IDs can be entered here,
+
+            filter_vessel_classes: A vessel class, or list of vessel classes to filter on. Both vessel names or IDs can be entered here,
+
+        # Returns
+        `VesselMovementsResult`, containing all the vessel movements matching the given search terms.
+
+
+        # Example
+        Let's search for all vessels that departed from `Rotterdam [NL]` on the morning of 1st December 2018.
+
+        ```python
+        >>> from vortexasdk import VesselMovements
+        >>> df = VesselMovements().search(
+            filter_time_min="2017-10-01T00:00:00.000Z",
+            filter_time_max="2017-10-01T00:10:00.000Z",
+            filter_origins='rotterdam'
+        ).to_df().head(2)
+        ```
+
+        |    | start_timestamp          | end_timestamp            |   vessel.imo | vessel.name   | vessel.vessel_class   | origin.location.country.label   | origin.location.port.label   | destination.location.country.label   | destination.location.port.label   |   cargoes.0.quantity | cargoes.0.product.grade.label   |
+        |---:|:-------------------------|:-------------------------|-------------:|:--------------|:----------------------|:--------------------------------|:-----------------------------|:-------------------------------------|:----------------------------------|---------------------:|:--------------------------------|
+        |  0 | 2017-09-30T15:30:27+0000 | 2017-10-03T01:46:06+0000 |  9.21091e+06 | ADEBOMI 3     | handysize             | Netherlands                     | Rotterdam [NL]               | Netherlands                          | Rotterdam [NL]                    |                  nan | nan                             |
+        |  1 | 2017-08-29T14:51:32+0000 | 2017-10-04T14:46:21+0000 |  9.64544e+06 | AEGEAN VISION | suezmax               | Netherlands                     | Rotterdam [NL]               | Singapore                            | Singapore [SG]                    |               852261 | High Sulphur                    |
+
+        [Vessel Movements Endpoint Further Documentation](https://docs.vortexa.com/reference/POST/vessel-movements/search)
+
+        """
+        geog = lambda x: convert_to_geography_ids(to_list(x))
+        corporation = lambda x: convert_to_corporation_ids(to_list(x))
+        ves = lambda x: convert_to_vessel_ids(to_list(x))
+        product = lambda x: convert_to_product_ids(to_list(x))
```

Comment:
`nit:` named lambdas are considered un-pythonic.  In this case I'd suggest just creating the variables you want to pass to the `params` dict here, directly, instead of using lambdas.